csm_web/scheduler/views/wordoftheday.py

```diff
@@ -0,0 +1,68 @@
+from rest_framework.exceptions import PermissionDenied
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+from django.utils import timezone
+import datetime
+import re
+
+from ..models import Student, SectionOccurrence, Attendance
+from .utils import get_object_or_error
+
+@api_view(['PUT'])
+def submit(request, section_occurrence_pk):

```

Comment:
Since word of the day is a field and not a resource itself, rename the file to section_occurrence.py and change this functions' name to submit_word_of_the_day.